packages/driver/src/cy/mouse.ts

```diff
@@ -567,6 +567,21 @@ export const create = (state: StateFunc, keyboard: Keyboard, focused: IFocused,
           return { skipClickEventReason: 'element was detached' }
         }
 
+        // Only send click event if element is not disabled.
+        // First find an parent element that can actually be disabled
+        const canBeDisabled = (el) => {
+          const actuallyDisabledElements = ['button', 'input', 'select', 'textarea', 'optgroup', 'option', 'fieldset']
+
+          return actuallyDisabledElements.indexOf($elements.getTagName(el)) > -1
+        }
+
+        const parentThatCanBeDisabled = $elements.findParent(mouseUpPhase.targetEl, canBeDisabled) || $elements.findParent(mouseDownPhase.targetEl, canBeDisabled)
+
+        // Then check if parent is indeed disabled
+        if (parentThatCanBeDisabled !== null && $elements.isDisabled(parentThatCanBeDisabled)) {
```

Comment:
Since `isDisabled` needs to be a `JQuery` object, we need to wrap the `HTMLElement` returned from `findParentThatCanBeDisabled`
```suggestion
        if (parentThatCanBeDisabled !== null && $elements.isDisabled($(parentThatCanBeDisabled))) {
```